webapp/components/sidebar_header.jsx

```diff
@@ -49,20 +75,21 @@ export default class SidebarHeader extends React.Component {
 
     render() {
         var me = this.props.currentUser;
-        var profilePicture = null;
-
         if (!me) {
             return null;
         }
 
-        if (me.last_picture_update) {
-            profilePicture = (
-                <img
-                    className='user__picture'
-                    src={Client.getUsersRoute() + '/' + me.id + '/image?time=' + me.last_picture_update}
-                />
-            );
+        let profilePictureSrc = Client.getUsersRoute() + '/' + me.id + '/image';
+        if (typeof me.last_picture_update === 'string') {
```

Comment:
You can actually use `Client.getProfilePictureUrl(id, lastPictureUpdate)` here and it'll automatically handle adding the time query parameter as necessary